network/netplay/netplay.c

```diff
@@ -1270,6 +1356,10 @@ void netplay_load_savestate(netplay_t *netplay, retro_ctx_serialize_info_t *seri
       netplay->other_frame_count = netplay->self_frame_count;
    }
 
+   /* If we can't send it to the peer, loading a state was a bad idea */
```

Comment:
This is the function that runs when we load a savestate from the UI. Presently there's no way for netplay to veto such a load, so if it happens on cores that don't support savestate transmission (thankfully a minority), things will desync. Perhaps load state veto should be next on my todo list, but for right now that text is accurate.